.cmake-format.yaml

```diff
@@ -0,0 +1,254 @@
+# find . \( -iname CMakeLists.txt -o -iname *.cmake \) -a -not -path "./submodules/*" | xargs cmake-format -c .cmake-format.yaml -i
```

Comment:
Ich würde noch `build` mit in die Ordner liste aufnehmen, und das `*.cmake` braucht anführungsstriche damit es auch unter zsh läuft.
Ich würde das etwas "ausführlicher" beschreiben, damit man es nicht so schnell übersieht
```suggestion
# Example usage to format *.cmake  files, excluding `build` and `submodules` folders:
#
#find . \( -iname CMakeLists.txt -o -iname "*.cmake" \) -a -not -path "./submodules/*" -a -not -path "./build/*" | xargs cmake-format -c .cmake-format.yaml -i
```

Reply:
I added some yaml for documentation purposes.
It has a one-liner for copy/paste
And a commented multi-liner for insight (with backticks-substitution-comments because of multi-line command).